batcher/aligned/src/main.rs

```diff
@@ -108,9 +108,9 @@ pub struct SubmitArgs {
     #[arg(name = "Private key", long = "private_key")]
     private_key: Option<String>,
     #[arg(
-        name = "Max Fee",
+        name = "Max Fee (ether)",
         long = "max_fee",
-        default_value = "1300000000000000" // 13_000 gas per proof * 100 gwei gas price (upper bound)
+        default_value = "0.0013ether" // 0.0000013ether per proof * 0.000001 gwei gas price (upper bound)
```

Comment:
you multiplied gas price * gas price. it is gas*gas price

Reply:
I changed the comment to use `ether` and `ether`. I am a little confused by what you mean by gas price * gas price can you elaborate?